src/app.ts

```diff
@@ -1,8 +1,8 @@
 import cors from '@koa/cors';
-import Router, { RouterContext } from '@koa/router';
+import type Router from '@koa/router';
```

Comment:
> If you’ve used Flow before, the syntax is fairly similar. One difference is that we’ve added a few restrictions to avoid code that might appear ambiguous.

This is a bit annoying.